utils/obsproc/RTOFSInSitu.h

```diff
@@ -0,0 +1,107 @@
+#pragma once
+
+#include <iostream>
+#include <netcdf>    // NOLINT (using C API)
+#include <regex>
+#include <string>
+#include <vector>
+
+#include "eckit/config/LocalConfiguration.h"
+
+#include <Eigen/Dense>    // NOLINT
+
+#include "ioda/Group.h"
+#include "ioda/ObsGroup.h"
+
+#include "NetCDFToIodaConverter.h"
+
+#include "rtofs/RTOFSDataFile.h"
+#include "rtofs/RTOFSOb.h"
+
+
+namespace gdasapp
+{
+
+
+class RTOFSInSitu:
+    public NetCDFToIodaConverter
+{
+ public:
+    explicit RTOFSInSitu(
+        const eckit::Configuration & fullConfig,
+        const eckit::mpi::Comm & comm):
+        NetCDFToIodaConverter(fullConfig, comm)
+    {
+        variable_ = "temperature";
```

Comment:
A variable is going to be "seaSurfaceTemperature"??

Reply:
the variable name should be `waterTemperature` which in the ioda terminolgy is insitu temperature.